### PR TITLE
Add postgres orm and services layer

### PR DESCRIPTION
**Pull request by Gabrielpanga**
Add postgres orm and services layer

### CODE REVIEW

File: src/config/initializers/database.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { Sequelize } from "sequelize";
+import { loadEnvVars } from './env_vars';
+
+loadEnvVars();
+
+const { DB_DATABASE = '', DB_USER = '', DB_PASSWORD = '' } = process.env;
+
+export const database = new Sequelize(DB_DATABASE, DB_USER, DB_PASSWORD, {
```
Comment (kaiomagalhaes) [inline src/config/initializers/database.ts]: Will we create a database connection every time we load this file?

Comment (Gabrielpanga) [reply src/config/initializers/database.ts]: No, it builds the database on start and thats it, it reference the result

Comment (kaiomagalhaes) [reply src/config/initializers/database.ts]: Unless we're initializing it from the config it shouldn't be in the initializers. Maybe move it to a `database` folder?

File: src/services/user.service.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
+
```
Comment (kaiomagalhaes) [inline src/services/user.service.ts]: this file should be under `src/services/db/user.db.service.ts`

this way it is easy to know what is a db service and the other ones

**Pull request merged**